src/bin/cargo-plumbing/cli.rs

```diff
@@ -1,18 +1,21 @@
 use cargo_plumbing::CargoResult;
 use clap::Parser;
 
+use crate::plumbing::locate_project;
+
 #[derive(Debug, Parser)]
 #[command(bin_name = "cargo")]
 #[command(styles = clap_cargo::style::CLAP_STYLING)]
 pub(crate) enum Command {
-    #[command(subcommand)]
-    Plumbing(crate::plumbing::Plumbing),
+    /// Locate the project manifest file (`Cargo.toml`)
+    #[command(name = "locate-project")]
```

Comment:
Overriding the name is redundant because clap will automatically convert `LocateProject` to `locate-project`